### PR TITLE
add a few new Quests

### PR DESCRIPTION
**Pull request by Dream-Master**
MV chemical Bath
Coolant
First Kanthal
<s>Compressed chest</s>

### CODE REVIEW

**Comment by miozune** [comment]
We already have a quest for Compressed Chest (No.1517)

**Comment by Dream-Master** [comment]
Ok good to know will remove it. I just not found it.

File: config/betterquesting/DefaultQuests.json [hunk 1]
```diff
@@ -320994,6 +320994,500 @@
...
+          "repeatTime:3": -1,
+          "globalShare:1": 1,
+          "questLogic:8": "AND",
+          "repeat_relative:1": 1,
+          "name:8": "§6§lCooling Kanthal",
+          "isGlobal:1": 0,
+          "lockedProgress:1": 0,
+          "autoClaim:1": 0,
+          "isSilent:1": 0,
+          "desc:8": "Hot Kanthal ingots can be cool down in a mv chemical bath."
```
Comment (GTNH-Afx237v7) [inline config/betterquesting/DefaultQuests.json:321028]: Must be: "Hot Kanthal ingots can be cooled down in an MV Chemical Bath."

File: config/betterquesting/DefaultQuests.json [hunk 2]
```diff
@@ -320994,6 +320994,500 @@
...
+          "repeatTime:3": -1,
+          "globalShare:1": 1,
+          "questLogic:8": "AND",
+          "repeat_relative:1": 1,
+          "name:8": "§6§lFirst Kanthal",
+          "isGlobal:1": 0,
+          "lockedProgress:1": 0,
+          "autoClaim:1": 0,
+          "isSilent:1": 0,
+          "desc:8": "You made your first Kanthal ingots. Its time to make some more to upgrade your Electric Blast Furnace."
```
Comment (GTNH-Afx237v7) [inline config/betterquesting/DefaultQuests.json:321119]: Must be: "It's time [...]"

File: config/betterquesting/DefaultQuests.json [hunk 3]
```diff
@@ -320994,6 +320994,500 @@
...
+          "repeatTime:3": -1,
+          "globalShare:1": 1,
+          "questLogic:8": "AND",
+          "repeat_relative:1": 1,
+          "name:8": "§6§lCoolant",
+          "isGlobal:1": 0,
+          "lockedProgress:1": 0,
+          "autoClaim:1": 0,
+          "isSilent:1": 0,
+          "desc:8": "Make your first coolant out of water or distilled water and lapis dust in a mv mixer."
```
Comment (GTNH-Afx237v7) [inline config/betterquesting/DefaultQuests.json:321216]: Must be: "[...] in an MV Mixer."

File: config/betterquesting/DefaultQuests.json [hunk 4]
```diff
@@ -320994,6 +320994,500 @@
...
+          "repeatTime:3": -1,
+          "globalShare:1": 1,
+          "questLogic:8": "AND",
+          "repeat_relative:1": 1,
+          "name:8": "Armor Molds",
+          "isGlobal:1": 0,
+          "lockedProgress:1": 1,
+          "autoClaim:1": 0,
+          "isSilent:1": 0,
+          "desc:8": "Why using plates if you can make your Armor pieces out of molten metal?"
```
Comment (GTNH-Afx237v7) [inline config/betterquesting/DefaultQuests.json:321407]: Must be:" Why use plates when you can make your armor pieces out of molten metal?"

File: config/betterquesting/DefaultQuests.json (hunk 3 above)
Comment (Dream-Master) [reply config/betterquesting/DefaultQuests.json:321216]: done

**Comment by Dream-Master** [comment]
> We already have a quest for Compressed Chest (No.1517)

removed

**Pull request merged**